ztp/kube-compare-reference/compare.sh

```diff
@@ -70,4 +70,98 @@ function compare_cr {
   return $status
 }
 
-compare_cr renderedv1 ../source-crs compare_ignore
+sync_cr() {
+    local rendered_dir=$1
+    local source_dir=$2
+    local exclusionfile=$3
+    local status=0
+
+    local -a renderedFiles
+    readarray -t renderedFiles < <(read_dir "$rendered_dir" | grep yaml)
+
+    local -a sourceFiles
+    readarray -t sourceFiles < <(read_dir "$source_dir" | grep yaml)
+
+    local -a excludedFiles
+    readarray -t excludedFiles < <(grep -v '^#' "$exclusionfile" | grep -v '^$')
```

Comment:
Maybe, for resilience:
```suggestion
    readarray -t excludedFiles < <(grep -v '^ *#' "$exclusionfile" | grep -v '^$')
```